types/react-window-infinite-loader/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for react-window-size 1.2
+// Project: https://github.com/bvaughn/react-window-infinite-loader/
+// Definitions by: Nivesh Ravindran <https://github.com/Nibblesh>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.5
+
+import { FC, Ref, ReactNode, Component } from "react";
+import { ListOnItemsRenderedProps, FixedSizeList, VariableSizeList } from 'react-window';
+
+type OnItemsRendered = (props: ListOnItemsRenderedProps) => any;
+
+interface InfiniteLoaderProps {
+    isItemLoaded: (index: number) => boolean;
+    loadMoreItems: (startIndex: number, stopIndex: number) => Promise<any>;
+    itemCount: number;
+    children: (props: {onItemsRendered: OnItemsRendered, ref: Ref<any>}) => ReactNode;
+}
+
+declare class InfiniteLoader extends Component<InfiniteLoaderProps> {
+}
+
+export default InfiniteLoader;
```

Comment:
This package was declared using `module.exports = ...` or some variant. It should be declared with an `export = ...` like one of the following:

### Variables

```ts
declare var xyz: SomeType;

export = xyz;
```

### Classes

```ts
class Foo {
  // ...
}

export = Foo
```

### Functions

```ts
function foo(): any;

export = foo;
```

To import the entity using a default import, you will need to enable `esModuleInterop`. Alternatively, you can just import using a more appropriate syntax like

```ts
import foo = require("foo");
```

[See more here in the FAQ](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9f9f6430117d025aef62ed56128deb65508edc86/README.md#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default).

Reply:
have updated to `export =`

thanks for the resource link and the info